src/mscorlib/shared/System/IO/StreamWriter.cs

```diff
@@ -963,14 +963,14 @@ private bool HaveWrittenPreamble_Prop
                 byte[] preamble = encoding.GetPreamble();
                 if (preamble.Length > 0)
                 {
-                    await stream.WriteAsync(preamble, 0, preamble.Length, cancellationToken).ConfigureAwait(false);
+                    await stream.WriteAsync(new ReadOnlyMemory<byte>(preamble), cancellationToken).ConfigureAwait(false);
```

Comment:
Do we need to explicitly create a ROM here? Would implicit cast work? I don't see an array based overload that has 2 parameters (second one being the cancellation token).

Reply:
> Do we need to explicitly create a ROM here?

Being explicit helps me when reading the code to know that I'm using the desired overload.  Otherwise I need to start paying attention to how many arguments there are and whether I might be slipping into the wrong one.